pkgconfig/meson.build

```diff
@@ -2,49 +2,58 @@ pkg = import('pkgconfig')
 
 vars = ['libva_version=' + libva_version, 'va_api_version=' + va_api_version]
 
-va_vars = vars + ['driverdir=' + driverdir]
-pkg.generate(libva,
-  description: 'Userspace Video Acceleration (VA) core interface',
-  filebase: 'libva',
-  variables: va_vars,
-  version: va_api_version)
-
-if WITH_DRM
-  drm_vars = vars + ['display=drm']
-  pkg.generate(libva_drm,
-    description: 'Userspace Video Acceleration (VA) DRM interface',
-    filebase: 'libva-drm',
-    requires: 'libva',
-    variables: drm_vars,
+if WITH_WIN32
+  va_vars = vars + ['driverdir=' + driverdir] + ['display=win32']
+  pkg.generate(libva,
+    description: 'Userspace Video Acceleration (VA) Win32 interface',
+    filebase: 'libva',
+    variables: va_vars,
     version: va_api_version)
```

Comment:
If I'm reading this correctly, this provides only a `libva.pc` which contains `display=win32` where every else is a `libva.pc`  + `libva-$platform.pc` where the latter has `display=$platform`.

Unless there is a very compelling reason to deviate I would suggest using the pre-existing approach. This means that:
 - there will be no if/else spaghetti here
 - there will be less if/else spaghetti in the user-code - ffmpeg or others